common/protob/messages-management.proto

```diff
@@ -121,8 +121,11 @@ message Features {
     optional uint32 display_rotation = 39;      // in degrees from North
     optional bool experimental_features = 40;   // are experimental message types enabled?
     optional bool busy = 41;                    // is the device busy, showing "Do not disconnect"?
-    optional HomescreenFormat homescreen_format = 42;   // format of the homescreen, 1 = TOIf 144x144, 2 = jpg 240x240
+    optional HomescreenFormat homescreen_format = 42;   // format of the homescreen, 1 = TOIf, 2 = jpg
     optional bool hide_passphrase_from_host = 43;   // should we hide the passphrase when it comes from host?
+    optional uint32 homescreen_width = 44;         // homescreen width in pixels
+    optional uint32 homescreen_height = 45;         // homescreen height in pixels
```

Comment:
Not sure about the arbitrary size screen thing. On one hand it's nice to see that the UI works reasonably well on display it was not developed for. On the other hand it would make sense to have this build as similar to model T as possible, and probably shouldn't add features to trezorlib that are not supported by anything we currently sell.

My proposal is to keep the 320x240 resolution but only support 240x240 homescreens for now, and revert the protobuf changes. Thoughts @matejcik?

Reply:
This is not strictly needed here and we can revert the changes now, but i expect that we will run into this sooner or later- e.g. T3W1 will need different homescreen size too and we will have to differentiate somehow, and adding Jpeg320x240 seemed like worse option to me